products/mozilla_org/test_mozilla_org.md

```diff
@@ -0,0 +1,66 @@
+# Testing Mozilla.org
+
+Mozilla.org is highly visible because the site houses the basic info of all Mozilla produced products, conveys Mozilla’s mission, vision, and values it stands for. Additionally, it promotes initiatives and campaigns in time of these events. As a result, new pages are created. New content is added to existing pages. New layout is introduced. The localized versions reach 60% of the Mozilla users globally. It is very important that, not only the main pages are localized, they are thoroughly tested before they are launched on production.  
+
+## Key Links
+* Production: https://www.mozilla.org/(locale)/
+* Staging: https://www-dev.allizom.org/(locale)/
+* Repository:  https://github.com/mozilla-l10n/www.mozilla.org/
+* Pontoon: https://pontoon.mozilla.org/projects/mozillaorg/
+* Locamotion: https://mozilla.locamotion.org/projects/mozilla_lang/
+* Web dashboard: https://l10n.mozilla-community.org/webdashboard/{local code}. Visit this page on a regular basis to check localization progress, pending work, deadline, and errors that were introduced during translation. 
+
+We highly advise you to ask other community members to conduct peer review not only on Pontoon or Locamotion, but on staging. While not all the languages are required for certain projects, each community can opt in the projects at a later time.
+
+## What to Test
+
+### Pre-L10n Test
+* Have your [glossary]((https://transvision.mozfr.org/) available as a reference, select mozilla.org as Repository, your language as your Target Locale
+* Associate the page with a product or site if it is localized. (e.g.: Firefox, Test Pilot)
```

Comment:
Not sure I understand this sentence. By associate, we mean “Make the content of the page consistent with product translations”?